### ISSUE TITLE
Narrator/NVDA is announcing the wrong row count: A11y_Maow_Follow a workshop_Prerequisites_Info and Relationships

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#117]
### GitHub Tags
#A11yTCS; #A11ySev2; #ChromiumEdge; #WCAG1.3.1; #Win11; #GH_Maow_Web_Mar24; #DesktopWeb; #Maow; #Info and Relationships; #AILimited: #Narrator; #NVDA; #ScreenReader; #A11yMAS;

### Actual Result:
Narrator/NVDA is announcing the wrong row count. 
### Expected Result:
Narrator/NVDA should announce the row count correct. It should announce 5 rows instead of 6 as there is only 5 row is available.

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/workshop/create-workshop/?step=0#
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL https://moaw.dev/workshop/create-workshop/
2. Create A Workshop screen will get open.
3. TAB till "Edit the workshop metadatap" section.
4. Verify that the Narrator/NVDA not announcing anything when we are searching in the search workshop search box.
### User Impact:
Screen reader will not know about the result user is getting after searching in the search box.

### Attachment:
**NVDA:**
![NVDA](https://github.com/microsoft/moaw/assets/124244478/ad232868-d521-4c6d-b276-96dd8c1f39b0)

**Narrator:**
![Narrator1](https://github.com/microsoft/moaw/assets/124244478/7a7bca71-a708-4a1a-9c11-d408e4adf5c5)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?
